method-README-template.md

```diff
@@ -1,65 +1,92 @@
+Provide a brief and exact description of the method clearly mentioning its purpose i.e., what the method does in or aims to achieve in abstract terms (avoiding technical details). Focus should be to emphasize on the functionality as a blackbox, helping researchers with different levels of expertise and experience to understand what the method is doing. Brielfly explain the input and output of the method and its note worthy features. Provide link(s) to related papers from the social science domain using the method or similar methods for solving social science research questions. 
+
+A second paragraph may be provided to highlight the reproducibility aspect of the method providing details or references to the resources used by the method, the data used in building the pre-trained modules. It should also provide different configuration parameters to assist in altering the behavior of the method for a related use case. 
+
+**Example:** *This method helps uncover hidden themes within a collection of text documents, making it a valuable tool for exploring unfamiliar domains. For example, a social scientist analyzing public discussions on social media or academic papers on a particular topic can use this method to identify recurring themes or topics. The method assumes that each document contains a mixture of topics and each topic comprises a distinct set of words. By processing a text file with one document per line, the method generates two key outputs i.e., i) the probability of each topic appearing in each document ii) the most representative words for each topic, along with their probabilities.*
+
+*This approach is built on [Latent Dirichlet Allocation (LDA)](https://www.jmlr.org/papers/volume3/blei03a/blei03a.pdf?ref=http://githubhelp.com), using a specialized technique called collapsed Gibbs sampling [(LDA with collapsed Gibbs sampling)](https://www.cs.cmu.edu/~wcohen/10-605/papers/fastlda.pdf). This enhances efficiency, producing a balanced topic distribution while allowing users control over the model’s internal workings.
+It uses [Markov chain Monti Carlo approach](https://en.wikipedia.org/wiki/Markov_chain_Monte_Carlo) to initialize the model with a random state. The method provides vanilla implementation (using only basic packages for loading data e.g., numpy, JSON, and random number generation) of Topic modeling with maximum control to customize its behavior. It gives users transparent control over internal decisions. The method is implemented as a class to extend its behavior easily.* 
 
 ## Keywords
-Few keywords placed after description highlighting the nature of the method 
+Provide 3 to 5 keywords that highlight purpose of the method and help in finding it (as a comma separated list).  
```

Comment:
If is a list, it should be a list! Lists can later be displayed as a comma separated list.